tests/test-sources/modules/output.nix

```diff
@@ -1,7 +1,91 @@
+{ pkgs, helpers, ... }:
+let
+  inherit (pkgs) lib;
+in
 {
   extraLuaPackages = {
     extraLuaPackages = ps: [ ps.jsregexp ];
     # Make sure jsregexp is in LUA_PATH
     extraConfigLua = ''require("jsregexp")'';
   };
+
+  # Test that all extraConfigs are present in output
+  all-configs.module =
+    { config, ... }:
+    let
+      configs = {
+        extraConfigLuaPre = "string.format('extraConfigLuaPre1')";
+        extraConfigLua = "string.format('extraConfigLua2')";
+        extraConfigLuaPost = "string.format('extraConfigLuaPost3')";
+        extraConfigVim = "let g:var = 'extraConfigVim4'";
+      };
+      configAssertion = name: config: [
```

Comment:
I think `name: value:` or `n: v:` is better, to avoid confusion with the overloaded `config` from the module args.

Reply:
Also lets name this something like `mkAssertion` or `assertFileContent` or `mkFileAssertion` (e.g.)